airflow/migrations/versions/30867afad44a_rename_concurrency_column_in_dag_table_.py

```diff
@@ -36,13 +36,20 @@
 
 def upgrade():
     """Apply Rename concurrency column in dag table to max_active_tasks"""
+    conn = op.get_bind()
+    is_sqlite = bool(conn.dialect.name == "sqlite")
```

Comment:
Why the additional `bool()` call?

Reply:
BTW this `conn.dialect.name` check appears much too often in our migrations files (not just those modified in this PR but generally) maybe we should extract the logic to somewhere like `airflow.utils.db`.

Reply:
> Why the additional bool() call? 

just a copy paste, no original work here :)

Reply:
removed `bool`